jenkins-pipelines/perf-regression-throughput-30min-with-hw-checks.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression.100threads.30M-keys.yaml","configurations/run_db_node_benchmarks.yaml"]''',
```

Comment:
Let's add this also to the performance_with_nemesis test.

Reply:
done